src/CommunityToolkit.Maui.Core/Primitives/PopupVisualOptions.macios.cs

```diff
@@ -4,17 +4,17 @@
 namespace CommunityToolkit.Maui.Core.Primitives;
 
 /// <summary>
-/// Visual Options for <see cref="CommunityToolkit.Maui.Core.Views.PopupView"/>
+/// Visual Options for <see cref="CommunityToolkit.Maui.Core.Views.AlertView"/>
 /// </summary>
-public class PopupViewVisualOptions
+public class AlertViewVisualOptions
```

Comment:
We also have to rename the file.
Since this is specific for macios, the name makes sense. Because on Android, alert (toast) doesn't support  corner radius

Reply:
Good catch! Done ✅